src/main/java/org/subethamail/smtp/server/NOPSessionLifecycleListener.java

```diff
@@ -0,0 +1,27 @@
+package org.subethamail.smtp.server;
+
+/**
+ * A {@link SessionLifecycleListener} that doesn't perform any real work
+ *
+ * @author Diego Salvi
+ */
+public final class NOPSessionLifecycleListener implements SessionLifecycleListener {
```

Comment:
Assuming we return to `SessionHandler` name, this could be called `AcceptAllSessionHandler` or `SessionHandlerAcceptAll`.